test/tests/functional/pbs_highreslog.py

```diff
@@ -44,6 +44,34 @@ class TestHighResLogging(TestFunctional):
     tm_micro_re = re.compile(
         r'(\d{2,2}/\d{2,2}/\d{4,4}\s\d{2,2}:\d{2,2}:\d{2,2}.\d{6,6})')
 
+    def validate_trace_job_lines(self, jid=None):
+        lines = self.server.log_lines(logtype='tracejob', id=jid, n='ALL')
+        if len(lines) > 1:
+            # Remove first line as it is not log line
+            lines = lines[1:]
+        for line in lines:
+            m = self.tm_micro_re.match(line)
+            if m is None:
+                # If this is accounting log, ignore it as
+                # Accounting log does not have high res logging
+                line = line.split()
+                _msg = 'Not Found high resolution time stamp in log'
+                self.assertFalse(len(line) >= 3 and
```

Comment:
I'm not sure i understand how this code is determining whether a line is an accounting log, can you please explain more? perhaps with an example?

Reply:
The account log lines have an 'A' in front of them in the tracejob output .